package/yast2-network.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Fri Jun  9 09:40:11 UTC 2017 - mfilka@suse.com
+
+- Moving network setup in AY into 1s stage
```

Comment:
currently not aware of any - the PBI contains no fate number 

@kobliha: is there any fate request?